webapp/src/main/java/org/opentestsystem/rdw/admin/model/GroupQuery.java

```diff
@@ -0,0 +1,65 @@
+package org.opentestsystem.rdw.admin.model;
+
+import java.util.Set;
+
+public class GroupQuery {
```

Comment:
Inspired by @malaffoon 's RdwImportQuery object in RDW_Ingest, I find it cleaner to pass around complex types rather than individual method params for a couple of reasons:

1.  It keeps our method parameter lists small
2.  Adding / removing parameters doesn't break the interface / method signatures.  Only need to update the complex type.